flocker/provision/_install.py

```diff
@@ -1393,6 +1401,10 @@ def configure_cluster(cluster, dataset_backend_configuration):
     :param dict dataset_backend_configuration: Configuration parameters to
         supply to the dataset backend.
     """
+    setup_action="start"
+    if(provider == "managed"):
+        setup_action="restart"
```

Comment:
Please put spaces around the assignment operator, e.g.`setup_action = "start"`).